sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/HyperLogLogPlusPlus.scala

```diff
@@ -167,7 +173,7 @@ case class HyperLogLogPlusPlus(
     val v = child.eval(input)
     if (v != null) {
       // Create the hashed value 'x'.
-      val x = MurmurHash.hash64(v)
+      val x = hasher.computeHash(v, childDataType)
```

Comment:
For the final version I want to create an xxHash companion object to the expression which implements this method.